content/learn/book/getting-started/ecs/_index.md

```diff
@@ -78,11 +78,11 @@ struct Name(String);
 We can then add `People` to our {{rust_type(type="struct" crate="bevy_ecs" name="World")}} using a "startup system". Startup systems are just like normal systems, but they run exactly once, before all other systems, right when our app starts. Let's use {{rust_type(type="struct" crate="bevy_ecs" name="Commands")}} to spawn some entities into our {{rust_type(type="struct" crate="bevy_ecs" name="World")}}:
 
 ```rs
-fn add_people(commands: &mut Commands) {
+fn add_people(mut commands: Commands) {
     commands
-        .spawn((Person, Name("Elaina Proctor".to_string())))
-        .spawn((Person, Name("Renzo Hume".to_string())))
-        .spawn((Person, Name("Zayna Nieves".to_string())));
+        .spawn_bundle((Person, Name("Elaina Proctor".to_string())))
+        .spawn_bundle((Person, Name("Renzo Hume".to_string())))
+        .spawn_bundle((Person, Name("Zayna Nieves".to_string())));
```

Comment:
```suggestion
    commands.spawn_bundle((Person, Name("Elaina Proctor".to_string())));
    commands.spawn_bundle((Person, Name("Renzo Hume".to_string())));
    commands.spawn_bundle((Person, Name("Zayna Nieves".to_string())));
```

Reply:
Politely disagree. I don't mind the style, but this is inconsistent with how commands are used in all of the other official examples.

Reply:
My point was that the `commands.` in the beginning were missing, not the spawn_bundle vs insert.